lib/i2cscan/i2cscan.cpp

```diff
@@ -1,11 +1,18 @@
 #include "i2cscan.h"
+#include "../../src/globals.h"
```

Comment:
Do you really need this include?

Reply:
yes, it is needed for `uint8_t portExclude[] = {LED_PIN};`